ci-operator/config/openshift/release/openshift-release-master__ci-4.16.yaml

```diff
@@ -209,41 +207,38 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-azure-sdn
-- as: e2e-azure-sdn-techpreview
+- as: e2e-azure-ovn-techpreview
   interval: 168h
   steps:
     cluster_profile: azure-2
     env:
       BASE_DOMAIN: ci2.azure.devcluster.openshift.com
       FEATURE_SET: TechPreviewNoUpgrade
-      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
     observers:
       enable:
       - observers-resource-watch
-    workflow: openshift-e2e-azure-sdn
+    workflow: openshift-e2e-azure-ovn
 - as: e2e-azure-sdn-serial
   interval: 168h
   steps:
     cluster_profile: azure-2
     env:
       BASE_DOMAIN: ci2.azure.devcluster.openshift.com
-      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
     observers:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-azure-sdn-serial
-- as: e2e-azure-sdn-techpreview-serial
+- as: e2e-azure-techpreview-serial
```

Comment:
Preference would be to keep the network type in the [job name](https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/) as some code still relies on parsing the job names